Sources/LiCore/Managers/LiAuthManager.swift

```diff
@@ -117,6 +118,7 @@ public class LiAuthManager: NSObject, InternalLiLoginDelegate {
         KeychainWrapper.standard.removeObject(forKey: LiCoreConstants.UserDefaultConstants.liLithiumUserId)
         KeychainWrapper.standard.removeObject(forKey: LiCoreConstants.UserDefaultConstants.liUserId)
         KeychainWrapper.standard.removeObject(forKey: LiCoreConstants.UserDefaultConstants.liTenantId)
+        UserDefaults.standard.removeObject(forKey: LiCoreConstants.UserDefaultConstants.liUserLoginStatus)
```

Comment:
The line is duplicated. Check line `113`.

Reply:
Will add the change in https://github.com/lithiumtech/li-ios-sdk/pull/37